azure-pipelines.yml

```diff
@@ -254,3 +254,27 @@ jobs:
       continueOnError: true
       condition: not(succeeded())
 
+- job: RichCodeNav_Indexing
+  pool:
+    name: NetCorePublic-Pool
+    queue: BuildPool.Windows.10.Amd64.VS2017.Open
+  strategy:
+    maxParallel: 2
+    matrix:
+      debug:
+        _configuration: Debug
+      release:
+        _configuration: Release
```

Comment:
I think it would be enough to do 1 configuration only (Debug likely to get more coverage)

Reply:
Yes I was just testing with the same setup from previous jobs so didn't make much modifications, and I removed those extra settings